webpack/react_app/redux/actions/job_invocations/index.js

```diff
@@ -0,0 +1,60 @@
+import { isNil } from 'lodash';
+
+import {
+  JOB_INVOCATIONS_GET_JOB_INVOCATIONS,
+  JOB_INVOCATIONS_POLLING_STARTED,
+} from '../../consts';
+
+import API from 'foremanReact/API';
+
+const defaultJobInvocationsPollingInterval = 10000;
+const jobInvocationsInterval = isNil(process.env.JOB_INVOCATIONS_POLLING) ?
```

Comment:
this seems to be repeating it self as a pattern, maybe it makes more sense to extract it to a common functions in core and reuse here and in notifications?

Reply:
Indeed - that's the kind of abstraction ActionCable provides for ws except we can't use it (yet).

Reply:
IMHO not just AC/WS - but code wise this is falling into the category of DRY